thunder/__init__.py

```diff
@@ -203,8 +203,18 @@ def __init__(self, model, compiled_model_call):
         super().__init__()
         self._model = model
 
+        del self.training
         self._forward_fn = compiled_model_call
 
+    def get_buffer(self, name):
+        return self._model.get_buffer(name)
+
+    def get_parameter(self, name):
+        return self._model.get_parameter(name)
+
+    def get_submodule(self, name):
+        return self._model.get_submodule(name)
```

Comment:
We might want to add logic to `__getattr__` so that if the method comes from `nn.Module` and is not overridden, we automatically route to `self._model`

Avoiding the need for these manual additions

Reply:
The idea here is that the next PR will change those to include overrides.